recipe/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
     - openblas 0.2.20|0.2.20.*
     - lapack
     - boost-cpp 1.66.0
-    - hdf5 1.10.1
+    - hdf5 
```

Comment:
Actually, all of the pins here should be removed

Reply:
the remaining ones are actually required...

Reply:
Actually, I think they are now covered in conda-forge-pinning.  It is a mistake to keep these.

Reply:
I am not sure I understand, in a docker image, if I remove the 1.66 and the 2.48 for `boost` and `glibmm` I can't compile `cyclus`...

Reply:
Oh really? That is bad...

Reply:
we depends on `boost/shared_ptr.hpp` that has been deprecated, and we have some symbol issue with `glibmm`
This need to be fix, but as I don't have the time right now...

Reply:
Ahhh ok, that makes sense